CM buildImage step failing due to missing dependency

**Describe the bug**
Cloud Manager builds for customers including assetshare.all-2.0.2 as a sub-package have recently started failing.

**Environment**
* AEM Version: Latest Cloud Manager version
* Asset Share Commons Version: 2.0.2
* Deploys for all environments

**To Reproduce**
Steps to reproduce the behavior:
1. Deploy assetshare.all-2.0.2.zip as a subpackage with a cloud manager deploy
3. buildImage step fails with this error
```
[ERROR] Error while assembling launcher: Package has unresolved dependencies: adobe/cq60:core.wcm.components.content:2.14.0
shaded.org.apache.jackrabbit.vault.packaging.DependencyException: Package has unresolved dependencies: adobe/cq60:core.wcm.components.content:2.14.0
	at shaded.org.apache.jackrabbit.vault.packaging.registry.impl.ExecutionPlanBuilderImpl.resolveInstall(ExecutionPlanBuilderImpl.java:257)
	at shaded.org.apache.jackrabbit.vault.packaging.registry.impl.ExecutionPlanBuilderImpl.validate(ExecutionPlanBuilderImpl.java:239)
	at org.apache.sling.feature.extension.content.ContentHandler.buildExecutionPlan(ContentHandler.java:87)
	at org.apache.sling.feature.extension.content.ContentHandler.handle(ContentHandler.java:128)
	at org.apache.sling.feature.launcher.impl.FeatureProcessor.prepareLauncher(FeatureProcessor.java:212)
	at org.apache.sling.feature.launcher.impl.Bootstrap.run(Bootstrap.java:160)
	at org.apache.sling.feature.launcher.impl.Main.main(Main.java:200)
Exception in thread "main" java.lang.IllegalStateException: The instance is not running
	at com.adobe.granite.composite.init.LogTail.checkInstanceIsAlive(LogTail.java:229)
	at com.adobe.granite.composite.init.LogTail.waitUntilInstanceIsInitialized(LogTail.java:88)
	at com.adobe.granite.composite.init.Main.main(Main.java:67)
```

**Expected behavior**
The buildImage step needs to pass.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- #607 - asset-share-commons.all package's dependency on Core Components breaks AEM CS deployment. Removing this hard dependency.
+
 ## [v2.0.2]
 
 ### Fixed

--- a/all/pom.xml
+++ b/all/pom.xml
@@ -54,6 +54,7 @@
                         <cloudManagerTarget>all</cloudManagerTarget>
                     </properties>
                     <finalName>asset-share-commons.all-${project.version}</finalName>
+                    <allowIndexDefinitions>true</allowIndexDefinitions>
                     <embeddeds>
                         <embedded>
                             <groupId>com.adobe.aem.commons</groupId>
@@ -74,15 +75,6 @@
                         </embedded>
                         <!-- Remove ui.contnet.sample from all package, as it should not be installed as part of real uses of Asset Share commons -->
                     </embeddeds>
-                    <allowIndexDefinitions>true</allowIndexDefinitions>
-                    <dependencies>
-                        <!-- Require AEM WCM Core Components to be installed on AEM
-                             Asset Share Commons will not auto-install Core Components for you -->
-                        <dependency>
-                            <groupId>com.adobe.cq</groupId>
-                            <artifactId>core.wcm.components.content</artifactId>
-                        </dependency>
-                    </dependencies>
                 </configuration>
             </plugin>
             <plugin>
@@ -189,12 +181,6 @@
             <artifactId>assetshare.ui.content.sample</artifactId>
             <version>${project.version}</version>
             <type>zip</type>
-        </dependency>
-        <dependency>
-            <groupId>com.adobe.cq</groupId>
-            <artifactId>core.wcm.components.content</artifactId>
-            <version>${core.wcm.components.version}</version>
-            <type>zip</type>
-        </dependency>
+        </dependency>>
     </dependencies>
 </project>

--- a/all/pom.xml
+++ b/all/pom.xml
@@ -181,6 +181,6 @@
             <artifactId>assetshare.ui.content.sample</artifactId>
             <version>${project.version}</version>
             <type>zip</type>
-        </dependency>>
+        </dependency>
     </dependencies>
 </project>

